quantum/process_keycode/process_unicode_common.c

```diff
@@ -78,14 +79,14 @@ void cycle_unicode_input_mode(int8_t offset) {
 void persist_unicode_input_mode(void) { eeprom_update_byte(EECONFIG_UNICODEMODE, unicode_config.input_mode); }
 
 __attribute__((weak)) void unicode_input_start(void) {
-    unicode_saved_caps_lock = host_keyboard_led_state().caps_lock;
+    unicode_saved_num_lock  = host_keyboard_led_state().num_lock;
```

Comment:
Hmm, looks like I mistakenly removed the caps lock line in my suggestion. My bad. This should fix it:

```suggestion
    unicode_saved_caps_lock = host_keyboard_led_state().caps_lock;
    unicode_saved_num_lock  = host_keyboard_led_state().num_lock;
```